Automattic/wp-calypso
# Change branding from WooCommerce Payments to WooPayments

Fixes https://github.com/Automattic/woocommerce-payments-server/issues/3526

## Proposed Changes

* Change branding from WooCommerce Payments to WooPayments in the Jetpack connect/authorize screen

## Testing Instructions

Checkout this branch of `wp-calypso` and run it locally (`yarn && yarn start`, add `127.0.0.1 calypso.localhost` to your `/etc/hosts`). Wait until you see `Ready! All assets are re-compiled. Have fun!`

Add these constants to your local WordPress site:
```
define( 'WOOCOMMERCE_CALYPSO_ENVIRONMENT', 'development' );
define( 'JETPACK__SANDBOX_DOMAIN', 'yourtestblog.wordpress.com' );
```

To start a Jetpack connection from WooCommerce Payments you need:
- Check out `develop` branch of https://github.com/automattic/woocommerce-payments, install it locally. 
- Launch the tube (`npm run tube:start` because Jetpack does not support redirect to the localhost).
- If you already have Jetpack connected, disconnect your site from Jetpack using https://github.com/Automattic/client-example. You’ll need to use Hard disconnect to disconnect WCPay.
- Visit payments connect page https://example.jurassic.tube/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fconnect and click Connect
- Hopefully, you'll be redirected to `calypso.localhost:3000/...`, and the screen will match exactly the one below.

## Screenshots

<img width="522" alt="image" src="https://github.com/Automattic/wp-calypso/assets/79862886/ad9b8e51-0cb7-427e-a787-cc45977a7db9">

## Pre-merge Checklist
